pdoc/templates/default/module.html.jinja2

```diff
@@ -547,6 +548,7 @@
         {{ decorators(cls) }}
         <span class="def">class</span>
         <span class="name">{{ cls.qualname }}</span>
+        <wbr>
```

Comment:
This introduces an unnecessary space between bases and qualname, please move it to the line above:

![image](https://user-images.githubusercontent.com/1019198/115713165-fe6d2900-a375-11eb-8d1a-15ba2ab7e955.png)